requirements.txt

```diff
@@ -18,16 +18,17 @@ Jinja2==2.11.3
 Mako==1.1.3
 MarkupSafe==1.1.1
 pipenv==2018.11.26
+psycopg2==2.7.7
```

Comment:
Sorry I forgot to remove this requirement, I installed it because I was getting a driver error from Flask-SQLAlchemy. But it was caused by me. for not Instantiating the cnf variable in the main __init__.py file.
I will remove this on the next commit.